src/main/kotlin/com/ampnet/payoutservice/exception/InternalException.kt

```diff
@@ -0,0 +1,7 @@
+package com.ampnet.payoutservice.exception
+
+class InternalException(val errorCode: ErrorCode, exceptionMessage: String) : Exception(exceptionMessage) {
+    companion object {
+        private const val serialVersionUID: Long = -2656013978175834059L
```

Comment:
What is the purpose of this value?

Reply:
This is used for Java's serialization mechanism. While we do not use it explicitly (and we shouldn't) it's a good practice to always define `serialVersionUID` for every class which implements `java.io.Serializable` - which is the case for exceptions since `Throwable` implements that interface.